game-rules.tex

```diff
@@ -3,28 +3,51 @@ \section{Game Rules}
 
 \begin{enumerate}
   \item The game, called \emph{\gamename}, is played in the arena defined in
-        Specification~\ref{spec:arena}. The objective is to collect containers
-        and deposit them in scoring zones.
-  \item The game is played by two teams, each team has two robots: a crane and
-        a forklift.
-  \item The forklift is used to pick up loose containers and move them to the
-        docking area. Each team's forklift starts in their starting area.
-  \item The arena contains 18 containers. The containers are labeled with
-        their owners: each team has 9 containers. One container from each
-        team is placed within the docking area, the others are outside the
-        docking area.
-  \item The crane is used to pick up a container from the docking area and
-        deposit it in a team's raised area.
-  \item At the end of the game, teams will be awarded points based on the
-        number of containers they collected, as follows:
-  \begin{enumerate}
-    \item For each container in the docking area, the team which owns the
-          container will be awarded \emph{1 point}.
-    \item For each container in the raised area, the team which owns the
-          container will be awarded \emph{3 points}.
-  \end{enumerate}
-  \item Containers are "in" a zone when their centre is within the zone.
+        Specification~\ref{spec:arena}. The objective is to collect tokens and
+        deposit them in scoring zones.
+  \item The arena contains 24 tokens.
+  \item There are nine scoring zones in the arena, arranged in a $3\times3$
+        grid. The central scoring zone is raised.
+  \item A scoring zone is considered to be `controlled' by a team if and only if:
+    \begin{enumerate}
+      \item the team has tokens in the zone.
+      \item no other team has a greater `weighting of tokens' in the zone.
+    \end{enumerate}
+  \item The `weighting of tokens' in a zone for a team is calculated from the sum of token weights in the zone.
+      Token weights are determined as follows:
+      \begin{enumerate}
+            \item A token `fully in' the zone has a weighting of $2$.
+            \item A token `partially in' the zone has a weighting of $1$.
+    \end{enumerate}
+  \item A `controlled region' is defined as follows:
+    \begin{enumerate}
+      \item A `controlled region' is a group of directly adjacent scoring zones controlled by the same team.
```

Comment:
Issue: A "controlled region" is defined here as being `>1` scoring zones. We should probably include that a single scoring zone is also considered.

Reply:
We should also define that a zone can only be in a single "controlled region". Otherwise a line of 3 scoring zones could be defined as a total of 6 different "controlled regions":

- 1
- 2
- 3
- 1,2
- 2,3
- 1,2,3

Reply:
```suggestion
      \item A `controlled region' is a contiguous set of scoring zones controlled by the same team.
```

Reply:
Fixed in https://github.com/sourcebots/smallpeice-rules/pull/21/commits/78ee2591bf06a1e72847e8ca91135ffaf09f3404